pkg/admission/pluginconfig_webhook.go

```diff
@@ -68,6 +68,11 @@ func DefaultPluginConfig(ctx context.Context, c client.Client, obj runtime.Objec
 		return err
 	}
 	pluginConfig.Spec.OptionValues = optionValues
+
+	// Default the ReleaseNamespace to the organization namespace if not set.
+	if pluginConfig.Spec.ReleaseNamespace == "" {
+		pluginConfig.Spec.ReleaseNamespace = pluginConfig.GetNamespace()
+	}
```

Comment:
This seems to contradict the validation that the field is immutable. I believe this validation can only be done once all `Plugins` have the `.spec.releaseNamespace` set

Reply:
True. The [below](https://github.com/cloudoperators/greenhouse/pull/98#discussion_r1555595741) addresses that. Or we introduce the immutability after the migration of existing instances since we have to touch the code again anyways

Reply:
Either way fine with me. Risk of this being manually changed seems quite low.